src/material-experimental/mdc-button/button.html

```diff
@@ -1,4 +1,4 @@
-<span class="mdc-button__ripple"></span>
+<span class="mdc-button__ripple mdc-fab__ripple"></span>
```

Comment:
should we conditionally apply these classes based on what type of button it is? i.e.

```html
<span
    [class.mdc-button__ripple]="!isFab"
    [class.mdc-fab__ripple]="isFab">
```

Reply:
added a `_isFab` property to button base